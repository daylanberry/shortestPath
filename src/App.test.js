import React from 'react';
import { shallow  } from 'enzyme'
import App from './App';
import renderer from 'react-test-renderer'


it('should return the correct length of component', () => {
  expect(shallow(<App />).length).toEqual(1)
})

it('should return the correct snapshot of App component', () => {
  const appComponent = renderer.create(<App/>).toJSON()
  expect(appComponent).toMatchSnapshot()
})