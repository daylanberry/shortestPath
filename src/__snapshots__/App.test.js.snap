// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the correct snapshot of App component 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="map"
    >
      <div
        className="row"
      >
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div>
               
              <img
                className="image"
                src="lake-tahoe.jpg"
              />
              F
            </div>
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div>
               
              <img
                className="image"
                src="legend.jpg"
              />
              E
            </div>
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
      </div>
      <div
        className="row"
      >
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
      </div>
      <div
        className="row"
      >
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div>
               
              <img
                className="image"
                src="https://i.pinimg.com/originals/93/c7/f7/93c7f77f60f2b27e3a32b179731b34a3.jpg"
              />
              D
            </div>
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div>
               
              <img
                className="image"
                src="downtown-Reno.jpg"
              />
              C
            </div>
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
      </div>
      <div
        className="row"
      >
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
      </div>
      <div
        className="row"
      >
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div>
               
              <img
                className="image"
                src="https://upload.wikimedia.org/wikipedia/en/5/55/UNR_Campus_North120520.jpg"
              />
              B
            </div>
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="blank"
            />
          </div>
        </ul>
        <ul
          className="cell"
        >
          <div
            onClick={[Function]}
          >
            <div
              className="image"
            >
              <img
                className="image"
                src="https://media.graytvinc.com/images/690*407/paddock+house+reno1.jpg"
              />
              A
            </div>
            <svg
              className="meeple"
            >
              meeple.svg
            </svg>
          </div>
        </ul>
      </div>
      <button
        onClick={[Function]}
      >
        Begin
      </button>
    </div>
  </div>
</div>
`;
